docs/develop/python/versioning.mdx

```diff
@@ -229,162 +249,9 @@ This video series examines into the behavior of the `patched()` function:
   </iframe>
 </div>
 
-#### Behavior When Not Replaying
-
-If the execution is not replaying, when it encounters a call to `patched()`, it first checks the event history.
```

Comment:
There's some useful content in here that maybe could go into a shared section for all core-based languages. It's not something people would commonly need to refer to, so I can certainly see removing it from here - but it can be useful for debugging and more complete understanding.